components/resources/wallet_strings.grdp

```diff
@@ -206,6 +206,7 @@
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_ETHEREUM_DESCRIPTION" desc="Add account modal create account Ethereum description">Supports EVM compatible assets on the Ethereum blockchain (ERC-20, ERC-721, ERC-1551, ERC-1155)</message>
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_SOLANA_DESCRIPTION" desc="Add account modal create account Solana description">Supports SPL compatible assets on the Solana blockchain</message>
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_FILECOIN_DESCRIPTION" desc="Add account modal create account Filecoin description">Store FIL asset</message>
+  <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_BITCOIN_DESCRIPTION" desc="Add account modal create account Bitcoin description">Bitcoin</message>
```

Comment:
are we sure this is what we want? This makes the section look like
```
 <LOGO> Bitcoin
         Bitcoin       <Create>
 ```

Reply:
@jamesmudgett 
What description for bitcoin do we want here?
![image](https://github.com/brave/brave-core/assets/168356/472eea31-30d6-4acc-ae6b-7299dd9f5235)